subsys/fs/fs_impl.h

```diff
@@ -28,6 +29,38 @@ extern "C" {
 const char *fs_impl_strip_prefix(const char *path,
 				 const struct fs_mount_t *mp);
 
+/**
+ * @brief Searches for file system type FS_* identifier, as defined in fs/fs.h,
+ *        compatible with string representation of type.
+ *
+ * @param string representation of type.
```

Comment:
This doesn't seem to name the parameter, which IMO ought to be something like `type_name` rather than `type_sz` which reads as "type size" since we don't use a "Hungarian" idiom for ASCIIZ values.

    @param type_name a text representation of the file system type